HPXMLtoOpenStudio/tests/test_hvac.rb

```diff
@@ -1114,6 +1114,18 @@ def test_heat_pump_advanced_defrost
     htg_coil = model.getCoilHeatingDXMultiSpeeds[0]
     _check_defrost(model, htg_coil, 0.0, 0.0, backup_fuel, 0.06667, 3444, 1055.1)
 
+    # Ductless heat pump w/ backup heat during defrost test
+    args_hash = {}
+    args_hash['hpxml_path'] = File.absolute_path(File.join(@sample_files_path, 'base-hvac-mini-split-heat-pump-ductless-backup-advanced-defrost-with-backup-heat-active.xml'))
+    model, _hpxml, _hpxml_bldg = _test_measure(args_hash)
+
+    # Get HPXML values
+    backup_fuel = EPlus.fuel_type(HPXML::FuelTypeElectricity)
+
+    assert_equal(1, model.getCoilHeatingDXMultiSpeeds.size)
+    htg_coil = model.getCoilHeatingDXMultiSpeeds[0]
+    _check_defrost(model, htg_coil, 10550.56, 1.0, backup_fuel, 0.06667, 3444, 1055.1)
```

Comment:
New test where we override the default and explicitly request backup heat during defrost.